packages/vue/src/components/organisms/SfProductCardHorizontal/SfProductCardHorizontal.vue

```diff
@@ -0,0 +1,301 @@
+<template>
+  <div class="sf-product-card-horizontal">
+    <component
+      :is="linkComponentTag"
+      :href="linkComponentTag === 'a' ? link : undefined"
+      :to="link && linkComponentTag !== 'a' ? link : undefined"
+      class="sf-product-card-horizontal__link sf-product-card-horizontal__link--image"
+    >
+      <div class="sf-product-card-horizontal__image-wrapper">
+        <slot name="image" v-bind="{ image, title }">
+          <template v-if="Array.isArray(image)">
+            <SfImage
+              v-for="(picture, key) in image.slice(0, 2)"
+              :key="key"
+              class="sf-product-card-horizontal__picture"
+              :src="picture"
+              :alt="title"
+              :width="imageWidth"
+              :height="imageHeight"
+            />
+          </template>
+          <SfImage
+            v-else
+            class="sf-product-card-horizontal__image"
+            :src="image"
+            :alt="title"
+            :width="imageWidth"
+            :height="imageHeight"
+          />
+        </slot>
+      </div>
+    </component>
+    <div class="sf-product-card-horizontal__main">
+      <div class="sf-product-card-horizontal__header">
+        <component
+          :is="linkComponentTag"
+          :href="linkComponentTag === 'a' ? link : undefined"
+          :to="link && linkComponentTag !== 'a' ? link : undefined"
+          class="sf-product-card-horizontal__link"
+        >
+          <slot name="title" v-bind="{ title }">
+            <h3 class="sf-product-card-horizontal__title">
+              {{ title }}
+            </h3>
+          </slot>
+        </component>
+        <slot name="price" v-bind="{ specialPrice, regularPrice }">
+          <SfPrice
+            v-if="regularPrice"
+            class="sf-product-card-horizontal__price"
+            :regular="regularPrice"
+            :special="specialPrice"
+          />
+        </slot>
+      </div>
+      <button
+        v-if="wishlistIcon !== false"
+        :aria-label="ariaLabel"
+        :class="wishlistIconClasses"
+        class="mobile-only"
+        @click="toggleIsOnWishlist"
+      >
+        <slot name="wishlist-icon" v-bind="{ currentWishlistIcon }">
+          <SfIcon
+            :icon="currentWishlistIcon"
+            color="black"
+            size="19px"
+            data-test="sf-wishlist-icon"
+          />
+        </slot>
+      </button>
+      <div class="sf-product-card-horizontal__details">
+        <slot name="description">
+          <p class="sf-product-card-horizontal__description desktop-only">
+            {{ description }}
+          </p>
+        </slot>
+        <slot name="configuration"></slot>
```

Comment:
```suggestion
        <slot name="configuration"/>
```
please add docs comment :)